src/webhook/mutation/pod/init_container_test.go

```diff
@@ -0,0 +1,25 @@
+package pod
+
+import (
+	"testing"
+
+	dtwebhook "github.com/Dynatrace/dynatrace-operator/src/webhook"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func TestCreateInstallInitContainerBase(t *testing.T) {
```

Comment:
this test doesn't seem to cover any none default case

Reply:
having a securityContext on the user pod and resource limits set on the dynakube is not default :)